### PR TITLE
docs: k6 executor docs

### LINKED ISSUE #1301: K6 Docs
Write down k6 examples

### PR DESCRIPTION
**Pull request by exu**
## Pull request description 

closes #1301

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: docs/executor-k6.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# K6 Performance Tests
+
+[K6](https://k6.io/docs/) Grafana k6 is an open-source load testing tool that makes performance testing easy and productive for engineering teams. k6 is free, developer-centric, and extensible.
+
+Using k6, you can test the reliability and performance of your systems and catch performance regressions and problems earlier. K6 will help you to build resilient and performant applications that scale.
+
+k6 is developed by Grafana Labs and the community.
+
+## Running a K6 test
```
Comment (jfermi) [inline docs/executor-k6.md:9]: ## Running a K6 Test

File: docs/executor-k6.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+K6 is integral part of Testkube. Testkube K6 executor is installed by default in recent Testkube installation. To run K6 test in Testkube you need to create Test.
```
Comment (jfermi) [inline docs/executor-k6.md:11]: K6 is integral part of Testkube. The Testkube k6 executor is installed by default during the Testkube installation. To run a k6 test in Testkube, you need to create a Test.

File: docs/executor-k6.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Using k6, you can test the reliability and performance of your systems and catch performance regressions and problems earlier. K6 will help you to build resilient and performant applications that scale.
+
+k6 is developed by Grafana Labs and the community.
+
+## Running a K6 test
+
+K6 is integral part of Testkube. Testkube K6 executor is installed by default in recent Testkube installation. To run K6 test in Testkube you need to create Test. 
+
+### Using files as input
```
Comment (jfermi) [inline docs/executor-k6.md:13]: ### Using Files as Input

File: docs/executor-k6.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+  http.get('https://kubeshop.github.io/testkube/');
+  sleep(1);
+}
+```
+
+Testkube and the K6 executor accepts a test file as an input.
```
Comment (jfermi) [inline docs/executor-k6.md:27]: Testkube and the k6 executor accepts a test file as an input.

File: docs/executor-k6.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+```sh
+kubectl testkube create test --file test.js --type "k6/script" --name k6-test
+```
+
+To run test just pass previously created test name:
```
Comment (jfermi) [inline docs/executor-k6.md:33]: To run a test, pass a previously created test name:

File: docs/executor-k6.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+kubectl testkube create test --file test.js --type "k6/script" --name k6-test
+```
+
+To run test just pass previously created test name: 
+
+```sh 
+kubectl testkube run test -f k6-test
+```
+
+You can also create Test based on Git repository:
```
Comment (jfermi) [inline docs/executor-k6.md:39]: You can also create a test based on a Git repository:

File: docs/executor-k6.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+```sh
+# create k6-test-script.js from this Git repository
+kubectl testkube create test --git-uri https://github.com/kubeshop/testkube-executor-k6.git --git-branch main --git-path examples --type "k6/script" --name k6-test-script-git
+```
+
+Testkube will clone repository and create Testkube Test Custom Resource in your cluster.
```
Comment (jfermi) [inline docs/executor-k6.md:46]: Testkube will clone the repository and create a Testkube Test Custom Resource in your cluster.

File: docs/executor-k6.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+You can also create Test based on Git repository:
+
+```sh
+# create k6-test-script.js from this Git repository
+kubectl testkube create test --git-uri https://github.com/kubeshop/testkube-executor-k6.git --git-branch main --git-path examples --type "k6/script" --name k6-test-script-git
+```
+
+Testkube will clone repository and create Testkube Test Custom Resource in your cluster. 
+
+### Using additional k6 arguments in your tests
```
Comment (jfermi) [inline docs/executor-k6.md:48]: Using Additional K6 Arguments in Tests

**Pull request merged**